policy-supplemental/AccessSensitiveInfoPolicy_SIAT.md

```diff
@@ -0,0 +1,13 @@
+| **Sensitive Information Access Table** 	|  	|
+|---	|---	|
+|  	|  	|
+| **Role/Individual** 	| **Accessible Sensitive Information** 	|
```

Comment:
Drop the bolding here, it's automatic for table headers.

Reply:
Done